components/feature/session/src/main/java/mozilla/components/feature/session/SessionUseCases.kt

```diff
@@ -72,20 +72,22 @@ class SessionUseCases(
          * no selected session a new session will be created using [onNoSession].
          *
          * @param url The URL to be loaded using the provided session.
-         * @param session the session for which the URL should be loaded.
+         * @param sessionId the ID of the session for which the URL should be loaded.
          * @param flags The [LoadUrlFlags] to use when loading the provided url.
          * @param additionalHeaders the extra headers to use when loading the provided url.
          */
         operator fun invoke(
             url: String,
-            session: Session? = sessionManager.selectedSession,
+            sessionId: String? = null,
```

Comment:
Just a heads-up that this will be a breaking change in Fenix in
https://github.com/mozilla-mobile/fenix/blob/07748f69b75144e8e247b742f66a756f999cb299/app/src/main/java/org/mozilla/fenix/home/intent/FennecBookmarkShortcutsIntentProcessor.kt#L47 

and 

https://github.com/mozilla-mobile/fenix/blob/7b1af41b4056da1ee2cb78a66232d962d7d1e07a/app/src/main/java/org/mozilla/fenix/customtabs/FennecWebAppIntentProcessor.kt#L72

Reply:
👍 
Will land now and then trigger a Nightly and prepare the Fenix PR.